add loader widget to confirmation box

dreacot opened the pull request:
Resolves #388 

The PR adds a loader widget to certain modals after the action button has been clicked

**Screenshots**

![Screenshot from 2021-05-04 21-45-51](https://user-images.githubusercontent.com/25265396/117067824-64689180-ad22-11eb-82c6-3d794204f6b7.png)

metaclips commented:
![Screenshot from 2019-08-26 18-27-23](https://user-images.githubusercontent.com/31141573/63712025-b0b4bc00-c834-11e9-89e7-0c66e5a402e5.png)
This are the issues noticed on your PR.

- Collapsible doesn't resize

-  Create new wallet gives invalid passphrase as shown on the top left.
Account page is already being worked on by a developer.

dreacot committed:
add loader to create new wallet modal

dreacot committed:
properly align the loader widget
add the loader widget to (CreateAccountTemplate, ChangePasswordTemplate, ChangeStartupPasswordTemplate, ImportWatchOnlyWalletTemplate)

dreacot committed:
add loader widget to send page
